Add Self-assessment Docs (#78)
kubernetes/sig-security: sig-security-assessments/README.md

# Self-assessment
A Self-assessment is a great starting point for Kubernetes subprojects to reach a security baseline. Our team 
will work with project maintainers create a threat model and assess the current security stance of all or a portion of 
the project. The resulting assessment will allow maintainers to determine gaps in their project's security and develop a
strategy to work toward improving it.

## Completed Assessments

> - [Cluster API](https://github.com/kubernetes/sig-security/blob/main/sig-security-assessments/cluster-api/self-assessment.md)

Reviewer: ```suggestion
- [Cluster API](https://github.com/kubernetes/sig-security/blob/1e6f14de2edae1662e2897a933071928515cc27c/sig-security-assessments/cluster-api/self-assessment.md)
```

## Getting Started
If you have a project for which you would like to complete a Self-assessment you can take a look at the 
[Self-assessment Recipe Book](https://github.com/kubernetes/sig-security/blob/main/sig-security-assessments/recipe-book.md)
for next steps.

## Getting Involved

> If you would like to participate in the SIG-Security Security Self-assessment subject please join the SIG-Security

Reviewer: Is this better?
```suggestion
If you would like to participate in the SIG Security Self-assessment area of work, please join the SIG Security
```

[mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-security) to be added to meeting invites.

## Leadership

### Subproject Lead
The subproject lead manages the operations of the subproject.

> * Ala Dewberry (**[@aladewberry](https://github.com/aladewberry)**), VMware

Reviewer 2: ```suggestion
* Ala Dewberry (**[@aladewberry](https://github.com/aladewberry)**): https://github.com/kubernetes/sig-security/blob/main/sig-security-assessments/OWNERS, VMware
```

For future us, would be easier to know if this needs an update as OWNERS file will be the source of truth on ownership of the subproject always.

Author: done!
